Bottom: 5634691a88a19120ba9dc0ad87a0e55b761812b6
Top:    e3e02282b753d2857d3847355a9435bc76129a87
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2008-10-02 16:47:15 +0100

Refresh of keep-all-pieces-of-a-divided-p

---

diff --git a/src/find.c b/src/find.c
index 9878dea..daf8ae2 100644
--- a/src/find.c
+++ b/src/find.c
@@ -2528,7 +2528,6 @@ IsLineInPolygon (LineTypePtr Line, PolygonTypePtr Polygon)
        return IsRectangleInPolygon (x1, y1, x2, y2, Polygon);
      }
   pg = Polygon->Clipped;
-  printf ("IsLineInPolygon hello world\n");
   do
     {
       if (Box->X1 <= pg->contours->xmax + Bloat
@@ -2536,12 +2535,8 @@ IsLineInPolygon (LineTypePtr Line, PolygonTypePtr Polygon)
           && Box->Y1 <= pg->contours->ymax + Bloat
           && Box->Y2 >= pg->contours->ymin - Bloat)
         {
-          printf ("got this far\n");
           if (!(lp = LinePoly (Line, Line->Thickness + Bloat)))
-            {
-              printf ("Error in ISLineInPolygon\n");
-              return FALSE;           /* error */
-            }
+            return FALSE;           /* error */
           return isects (lp, Polygon, True);
         }
     }
